malboxes/profiles/snippets/provision_powershell.json

```diff
@@ -4,8 +4,9 @@
 			{% if not windows_updates == "true" %}"{{ dir }}/scripts/windows/disable_auto-updates.ps1",{% endif %}
 			{% if not windows_defender == "true" %}"{{ dir }}/scripts/windows/disable_defender.ps1",{% endif %}
 			"{{ dir }}/scripts/windows/vmtools.ps1",
-			"{{ dir }}/scripts/windows/malware_analysis.ps1",
-			"{{ dir }}/scripts/windows/installtools.ps1"
+			"{{ dir }}/scripts/windows/installtools.ps1",
+			{% if customization_profile %}"{{ config_dir }}/scripts/user/{{ customization_profile }}.ps1",{% endif %}
+			"{{ dir }}/scripts/windows/malware_analysis.ps1"
```

Comment:
This change will need to be repeated in the new `provision_powershell_win7.json` file